src/css/style.scss

```diff
@@ -1,3 +1,170 @@
-h1 {
-    color: red;
-}
\ No newline at end of file
+* {
+    margin: 0; 
+    padding: 0;
+    box-sizing: border-box;
+}
+
+@font-face {font-family: "TwCenW01-Medium"; src: url("//db.onlinewebfonts.com/t/c862d8098ce50a7dc042ebbb9b94bbc8.eot"); 
+src: url("//db.onlinewebfonts.com/t/c862d8098ce50a7dc042ebbb9b94bbc8.eot?#iefix") 
+format("embedded-opentype"), url("//db.onlinewebfonts.com/t/c862d8098ce50a7dc042ebbb9b94bbc8.woff2") 
+format("woff2"), url("//db.onlinewebfonts.com/t/c862d8098ce50a7dc042ebbb9b94bbc8.woff") 
+format("woff"), url("//db.onlinewebfonts.com/t/c862d8098ce50a7dc042ebbb9b94bbc8.ttf") 
+format("truetype"), url("//db.onlinewebfonts.com/t/c862d8098ce50a7dc042ebbb9b94bbc8.svg#TwCenW01-Medium") format("svg"); } 
+
+
+//Variables
+$grey: #f1f1f1;
+
+
+//Basic styles
+html {
+    background: $grey;
+    font-family: "TwCenW01-Medium";
+}
+
+.container {
+    width: 90vw;
+    margin: auto;
+}
+
+button {
+    outline: none;
+    border: none;
+    cursor: pointer;
+    &:hover {
+        border-bottom: 1px solid black;
+    }
+}
+
+
+//Header
+.navbar {
+    height: 20vh;
+    width: 100%;
+    background: $grey;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-weight: 800;
+    font-size: 1.7em;
+    letter-spacing: 2px;
+    line-height: 1.2;
+    border-bottom: 1px solid black;
+    transition: all .4s ease;
+    z-index: 1;
+}
+
+.title {
+    font-family: 'GermanoRegular';
+    font-weight: normal;
+    font-style: normal;
+}
+
+//shrink navbar on scroll
+.shrink {
+    position: fixed;
+    font-size: 1em;    
+    height: 10vh;
```

Comment:
You need to ensure this scroll nav transition on scroll is smooth. By replacing between `fixed` and `relative` the code will jump around each time it changes. My suggestion: Start with `fixed` by default and adjust the rest of the content based on it ;)

Reply:
Yeah I have no idea what the difference is between e.g. using vh vs. rem in this case, I kinda used it arbitrarily 

> Start with fixed by default and adjust the rest of the content based on it ;)

adjust in what way? What exactly needs fixing, the sudden transition when you start scrolling or..?

Reply:
`vh` changes based on the browser height. So fix the browser is small, the header height will be smaller.
`rem` adjusts based on the user zoom settings. The normal is 16px, so that does not change that much.

> adjust in what way? What exactly needs fixing, the sudden transition when you start scrolling or..?

Let me put it in another way: Don't animate the position. Start with `fixed` by default and let it be even when scrolled ;)